Administrator Guide/Split brain and ways to deal with it.md

```diff
@@ -0,0 +1,68 @@
+# Split brain and the ways to deal with it
+
+### Split brain:
+Split brain is a situation where two or more replicated copies of a file become divergent. When a file is in split brain, there is an inconsistency in either data or metadata of the file amongst the bricks of a replica and do not have enough information to authoritatively pick a copy as being pristine and heal the bad copies, despite all bricks being up and online. For a directory, there is also an entry split brain where a file inside it can have different gfid/file-type across the bricks of a replica. Split brain can happen mainly because of 2 reasons:
+1. Due to network disconnect:
+Where a client temporarily loses connection to the bricks.
+2. Gluster brick processes going down or returning error:
+    - Server1 is down and server2 is up: Writes happen on server 2.
+    - Server1 comes up, server2 goes down (Heal not happened / data on server 2 is not replicated on server1): Writes happen on server1.
+    - Server2 comes up: Both server1 and server2 has data independent of each other.
+
+If we use the replica 2 volume, then there are high chances of ending up in split-brain.

```

Comment:
I think the problem is not that we will end up in split-brain. We can end up in split-brain with replica 3 as well. The problem with replica 2 is that we can't prevent a split-brain without losing availability. Quorum kind of becomes useless because of that.